### ISSUE TITLE
program unusable at 1280x720 resolution

### ISSUE THREAD

**Issue by jim-perkins** [keepassxreboot/keepassxc#4878]
## Steps to Reproduce
1. 60 inch LED tv from year 2011 with max resolutions of 1920x1080 running at 1280x720 to keep fonts readable at a 10ft viewing distance using wireless keyboard and mouse.
2.  ubuntu linux 18.04 with all updates
3. set display resolution at 1280x720
4. apt install keepassxc .  Which install version 2.3.1
5.  Install keepassxc-browser for chromium
6.  keepassxc-browser states "You are using an old version of KeePassXC.
Please download the latest version from keepassxc.org."
7.  sudo add-apt-repository ppa:phoerious/keepassxc ; sudo apt upgrade
8. keypassxc updated to 2.5.4-1ppa1~bionic1
9. open keepassxc
10.  size of program is so large and cant be reduced to fit on screen

Screenshot is similar to the ones where people complaining about high dpi screens.  This is just a regular 1080P tv that is over 9 years old.  can supply screenshot if requested

## Expected Behavior
Being able to update a program and it is usable

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
program is so large it goes off the screen and is unusable

[NOTE]: # ( Give us any additional information you may have. )
Thanks for your work on this project.  It is helping me move away from online stored passwords

KeePassXC - VERSION
see above

[NOTE]: # ( Pick choices based on your environment )

ubuntu 18.04 , xorg, budgie-desktop

**Comment by jim-perkins** [comment]
Thanks for the reply.  Unfortunately that made the fonts very very small. 

I have seen other, unrelated to this app, issues with 1280x720.  usually when I go into an advanced setting.  It just wont fit on the screen.  My laptop is also 1280x720.  

As a test  I set my resolution at 1920x1080 , without the env variable change, and ran keepassxc.  It didnt fit on the screen either.  Just wanted to let you know on a standard 1080p monitor the program is not usable as shipped , in my application.  Other variables: I do run the HDMI path thru a surround sound receiver so maybe I need to investigate that route.  Also have a discrete video card that possibly does other resolutions that the monitor can not display.

Ubuntu used to offer font scaling and I had more control over the dpi and could run at full resolution.  It appears they removed that option.  

I guess I will search out more general solutions to my display problems.  Any additional help is appreciated.  Thanks again for your work on this program

**Comment by droidmonkey** [comment]
You can use any value in that env var, try 0.75 or 0.8.

**Comment by jim-perkins** [comment]
thanks for the follow up. Found a solution that will work for me 

Tested a live install of ubuntu 20.04 which installed keepassxc 2.4.3 and the program was correctly sized for 1920x1080 and 1280x720 .  I then added the PPA and ugraded to 2.5.4 and it still looked good.  The only difference is that I havent loaded the nvidia display drivers but I doubt that will matter.  Did not have to add an environment variable.

Not sure if this was part of the problem but:
Ubuntu 20.04  utilized Qt 5 core module 5.12.8
Ubuntu 18.04 utilized Qt 5 core module 5.95

So finally, what will work for me is to upgrade Ubuntu from 18.04 to 20.04
thanks again
